### DISCUSSION TITLE
hide the "Tell us what you want to use Argo for" and version modals?

### DISCUSSION THREAD

**Discussion by trondhindenes** [argoproj/argo-workflows#9589] [Q&A] [answered]
I get the "Tell us what you want to use Argo for" box everytime I log in to Argo workflows, as well as the "what's new" version box. These are both annoying. Is there a way to properly dismiss them so they don't reappear?

**Comment by terrytangyuan** [comment, marked as answer]
Check out these environment variables (`*_MODAL`) for Argo Server: https://github.com/argoproj/argo-workflows/blob/master/docs/environment-variables.md#argo-server

**Reply by trondhindenes** [reply]
awesome, thanks!
